rezerwacje-front/src/views/Recruitments/components/RecruitmentCycleFormDialog/validateRecruitmentCycleForm.js

```diff
@@ -0,0 +1,12 @@
+export default function validateRecruitmentCycleForm(values) {
```

Comment:
You should also validate the dates (check if startTime <= endTime and compare them to start and end dates of the recruitment).

Reply:
It's not critical as we have backend validation checking this condition, so adding recruitment period with startTime > endTime is impossible